### DISCUSSION TITLE
Lesson 6 question: Installing Foundry (using a custom file path)

### DISCUSSION THREAD

**Discussion by dezlymacauley** [Cyfrin/foundry-full-course-cu#91] [Q&A] [answered]
To give some context:
I'm using a Windows Terminal with Ubuntu.
I used the cd command to change the file path to a folder that is stored on my external drive

My terminal after I have changed the directory:
```css
dezlymacauley@DezlyMacauley:/mnt/z/Dezly Macauley$
````
Next I ran this command to install foundry
```css
dezlymacauley@DezlyMacauley:/mnt/z/Dezly Macauley$ curl -L https://foundry.paradigm.xy
```

I get  this message:
```
Detected your preferred shell is bash and added foundryup to PATH. Run **'source /home/dezlymacauley/.bashrc'** 
or start a new terminal session to use foundryup.
Then, simply run 'foundryup' to install Foundry.
```
Should I proceed with:  **source /home/dezlymacauley/.bashrc'**

Or should I change it to match my file path? **source/z/Dezly Macauley.bashrc**

**Comment by cromewar** [comment, marked as answer]
Hello @DezlyMacauley 
thanks for your question, well the `.bashrc` and `.zshrc` are the files linus/mac uses to determine what the env variables are, so they will be always stored on the `/home/` path on this OS, please do not change this or your terminal will collapse.

Additionally I see you are storing your projects on the `/mnt/z` drive, which is ok... but slow. The translation layer on the file management between `wsl` and and windows is not that good, I would strongly suggest just to create a `Development` folder on your `wsl` home and store your projects, there. Is simply faster.

**Reply by dezlymacauley** [reply]
Thanks! I've followed your suggestion and created a folder in the home directory (rather than trying to use the /mnt/z) drive.
When I used the /mnt/z drive I got the speed warning from VS Code and ran into some path errors when using Foundry. 
So I decided to just use the default home directory and save myself any potential headaches down the line. 😅
